src/elements/CheckboxGroup.vue

```diff
@@ -146,7 +146,7 @@ export default {
 <template>
 <div>
   <Heading
-    font-family="museo"
+    font-family="special"
```

Comment:
Only because, in this example in this particular component, the Museo font was used to show how you could change the normal font of the element to something different, so the normal font wouldn't accomplish that. The string `special` is used instead of the variable `$font-special`, because we're in the CSS but the HTML, in which, I don't believe the tokens are defined. In the JS, the string is used to call the font's alias, as defined in `src/styles/_font-faces.scss`, in which the `$font-special` variable is used.

Reply:
Okay, this makes sense.  I assume that when HB-5101 is complete and the props accept the tokens we will be able to use the `$font-special` token here